back/src/api/common/services/alcohol/alcoholTaxCalculator.ts

```
@@ -0,0 +1,140 @@
+  private static readonly TYPE_MAPPING = {
+    strongAlcohol: 'alcoholStrong',
+    softAlcohol: 'alcoholWeak',
+    beer: 'beer',
+    wine: 'wine',
+    sparklingWine: 'wine',
+    spiritDrink: 'alcoholStrong',
+    alcoholIntermediate: 'alcoholWeak',
+  } as const;
+
+  private static mapType(type: string): string {
+    return this.TYPE_MAPPING[type as keyof typeof this.TYPE_MAPPING] || type;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)